dnn/activationfunction.py

```diff
@@ -3,92 +3,87 @@
 
 import numpy as np
 
+e = np.e
+
 ## 活性化関数
 # sigmoid関数
 def non_universal_sigmoid(x):
-    # avoid overflow
     sigmoid_range = 34.538776394910684
     if x <= -sigmoid_range:
         return 1e-15
     elif x >= sigmoid_range:
         return 1.0 - 1e-15
     return 1.0 / (1.0 + np.exp(-x))
 
-# vectorize non universal sigmoid
 def sigmoid(x):
     sig = np.vectorize(non_universal_sigmoid)
     return sig(x)
 
-# msigmoid(一次元配列 x)
-def msigmoid(x):
-    ans = x
-    node = len(x)
-    if node == 1:
-        ans[0] = sigmoid(x[0])
-        return ans
-    for i in range(1, node):
-        ans[i] = sigmoid(x[i])
-        return ans
-
 # tanh関数
 def tanh(x):
-    e = np.e
     return (e**x - e**-x) / (e**x + e**-x)
 
-# mtanh(一次元配列　x)
-def mtanh(x):
-    ans = x
-    node = len(x)
-    for i in range(1, node):
-        ans[i] = tanh(x[i])
-        return ans
+#identity関数
+def non_universal_identity(x):
+    identity_range = 1e+5
+    if x < -identity_range:
+        return -1e+5
+    elif x > identity_range:
+        return 1e+5
+    return x
+
+def identity(x):
+    iden = np.vectorize(non_universal_identity)
+    return iden(x)
+
+def non_universal_swish(x):
+    swish_range = 1e+5
+    if x < -identity_range:
+        return -1e+5
+    elif x < identity_range:
+        return 1e+5
+    return x / (1 + e ^ -x)
+
+def swish(x):
+    swi = np.vectorize(non_universal_swish)
+    return swi(x)
 
 # relu関数
-def relu(x):
+def non_universal_relu(x):
     if x < 0:
         return 0
     elif x > 1e+5:
         return 1e+5
-    else:
+    return x
```

Comment:
Actually, this don't work well for overflow. Regardless of this, I'll approve.(Just a comment!)